### PR TITLE
Add ability to send CV to the lever api

### PR DESCRIPTION
**Pull request by mobinzk**
Changed the post method to use `multipart/form-data` as the api only accepts files via `multipart/form-data`.

### CODE REVIEW

File: ViewModels/LeverPostingApplyViewModel.cs [hunk 1]
```diff
@@ -59,6 +62,75 @@ public string ToJson
             }
         }
 
+        public MultipartFormDataContent ToDataFrom()
```
Comment (peterkeating) [inline ViewModels/LeverPostingApplyViewModel.cs]: Rename this to `ToFormData`.

File: ViewModels/LeverPostingApplyViewModel.cs [hunk 2]
```diff
@@ -14,6 +16,7 @@ public class LeverPostingApplyViewModel
         public string Comments { get; set; }
         public Dictionary<string, string> Consent { get; set; }
         public string Email { get; set; }
+        public IFormFile Resume { get; set; }
```
Comment (peterkeating) [inline ViewModels/LeverPostingApplyViewModel.cs]: Reposition to respect alphabetical ordering of properties.

File: ViewModels/LeverPostingApplyViewModel.cs [hunk 3]
```diff
@@ -59,6 +62,75 @@ public string ToJson
             }
         }
 
+        public MultipartFormDataContent ToDataFrom()
+        {
+            var httpContent = new MultipartFormDataContent
+            {
+                // Double qoutes are required for keys as the Lever API doesn't accept the key without double qoutes.
+                // Solution from: https://stackoverflow.com/a/21874413
+                { new StringContent(!string.IsNullOrEmpty(Comments) ? Comments : string.Empty), @"""comments""" },
```
Comment (peterkeating) [inline ViewModels/LeverPostingApplyViewModel.cs]: I'd suggest moving the double quotes to it's own method (e.g. `FormatForLever`) as it's being used everywhere.

File: ViewModels/LeverPostingApplyViewModel.cs [hunk 4]
```diff
@@ -59,6 +62,75 @@ public string ToJson
...
+                { new StringContent(!string.IsNullOrEmpty(Email) ? Email : string.Empty), @"""email""" },
+                { new StringContent(!string.IsNullOrEmpty(Name) ? Name : string.Empty), @"""name""" },
+                { new StringContent(!string.IsNullOrEmpty(Org) ? Org : string.Empty), @"""org""" },
+                { new StringContent(!string.IsNullOrEmpty(Phone) ? Phone : string.Empty), @"""phone""" }
+            };
+
+            if (Urls != null && Urls.Any())
+            {
+                foreach (var item in Urls)
```
Comment (peterkeating) [inline ViewModels/LeverPostingApplyViewModel.cs]: You can shorten this by changing it to `Urls.Where(x => x.Value != null)`.

File: ViewModels/LeverPostingApplyViewModel.cs (hunk 1 above)
Comment (peterkeating) [inline ViewModels/LeverPostingApplyViewModel.cs]: SoundCloud isn't a fan of the length of this method (it's double what it recommends). Happy to chat through ideas on how we can reduce the length of it.

**Review by peterkeating — CHANGES_REQUESTED** [review]

**Comment by peterkeating** [comment]
@mobinzk You requested a review but I don't see any changes.

**Comment by mobinzk** [comment]
> @mobinzk You requested a review but I don't see any changes.

I've pushed up my changes but they don't come here!

**Review by mobinzk — COMMENTED** [review]
Perfect, thank you.

**Pull request merged**